.github/workflows/build_and_test_dbt.yaml

```diff
@@ -82,6 +82,6 @@ jobs:
         shell: bash
 
       - name: Move dbt manifest directory to update cache
-        run: mv "$TARGET_DIR" "$STATE_DIR"
+        run: rsync -av --delete "$TARGET_DIR/" "$STATE_DIR"
```

Comment:
[The error in question](https://github.com/ccao-data/data-architecture/actions/runs/5906036003/job/16021252615):

```
Run mv "$TARGET_DIR" "$STATE_DIR"
mv: cannot move 'target' to 'state/target': Directory not empty
Error: Process completed with exit code 1.
```

An alternative would be to `rm -Rf "$STATE_DIR" && mv "$TARGET_DIR" "$STATE_DIR"`, but I liked the one-liner suggested by [this StackExchange answer](https://superuser.com/a/623846).